src/components/Extension/ExtensionBadge.js

```diff
@@ -0,0 +1,12 @@
+import { Component } from 'react'
+
+class ExtensionBadge extends Component {
+  render () {
+    if (chrome && chrome.browserAction) {
+      chrome.browserAction.setBadgeText({ text: this.props.text })
+    }
+    return null
```

Comment:
This is a "renderless" component. Doesn't draw anything on the page, instead uses the chrome API to update the browser icon.